### DISCUSSION TITLE
rshell and ESP32:  no such file or directory

### DISCUSSION THREAD

**Discussion by A622266** [micropython/micropython#9441] [Hardware & Peripherals] [answered]
I have a new ESP32.  I have erased the flash and loaded esp32-20220618-v1.19.1.bin using esptool as described in the documentation.  These complete successfully.

I start rshell, and get the following output:

Using buffer-size of 32
Connecting to /dev/ttyUSB0 (buffer-size 32)...
Trying to connect to REPL  connected
Retrieving sysname ... esp32
Testing if ubinascii.unhexlify exists ... Y
Retrieving root directories ... /boot.py/
Setting time ... Sep 27, 2022 20:13:47
Evaluating board_name ... pyboard
Retrieving time epoch ... Jan 01, 2000
Welcome to rshell. Use Control-D (or the exit command) to exit rshell.

However, when I issue the command:
ls /flash

I get:
Cannot access '/flash': No such file or directory

Can anyone see an issue, or give some advice?
Thanks
Andy

**Comment by davefes** [comment, marked as answer]
Try:
```py
cd /pyboard
```

**Comment by peterhinch** [comment]
This is right. If you issue `ls /` in rshell you will see that the flash is mounted as `/pyboard`.
